.gitignore

```diff
@@ -40,6 +40,7 @@ dist/
 /screenshot/index.js.map
 /screenshot/package.json
 /screenshot/pixel-match.js
+/screenshot/pixel-match.js.map
```

Comment:
For my own knowledge, what was the cause of adding this to the `.gitignore`?

Reply:
Just if you build with the `DEBUG=true` set then this `.js.map` file is written to disk, it wasn't previously so it needs to be added